<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetToPriceGuideDialog</class>
 <widget class="QDialog" name="SetToPriceGuideDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set To Price Guide</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The prices of all selected items will be set to Price Guide values.&lt;br /&gt;&lt;br /&gt;Select which part of the price guide should be used:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="w_type_time"/>
     </item>
     <item>
      <widget class="QComboBox" name="w_type_price"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="w_advanced_button">
     <property name="styleSheet">
      <string notr="true">QToolButton {  border: none; backgroundColor: &quot;transparent&quot; }
</string>
     </property>
     <property name="text">
      <string>Advanced options</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::UpArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="w_advanced" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>26</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Only use these options if you know what you are doing!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="w_force_update">
        <property name="text">
         <string>Download even if already in cache.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="w_buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>w_buttons</sender>
   <signal>accepted()</signal>
   <receiver>SetToPriceGuideDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_buttons</sender>
   <signal>rejected()</signal>
   <receiver>SetToPriceGuideDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
